python/ray/util/sgd/v2/session.py

```diff
@@ -0,0 +1,116 @@
+import queue
+import time
+import threading
+
+from ray.util.sgd.v2.constants import TIME_THIS_ITER_S
+
+
+class Session:
+    """Holds information for training on each worker."""
```

Comment:
nit: Following offline discussion, maybe we should rename this to something more descriptive.

Reply:
As discussed on Slack, leaving as is.